docs/community/release_guide.md

```diff
@@ -0,0 +1,291 @@
+
+- Generate a pre-release note from GitHub for the subsequent voting.
+
+Goto the [release page](https://github.com/apache/celeborn/releases) and click the "Draft a new release" button, then it would jump to a new page to prepare the release.
+
+Filling in all the necessary information required by the form. And in the bottom of the form, choose the "This is a pre-release" checkbox. Finally, click the "Publish release" button to finish the step.
+
+> Note: the pre-release note is used for voting purposes. It would be marked with a **Pre-release** tag. After all the voting works(dev and general) are finished, do not forget to inverse the "This is a pre-release" checkbox. The pre-release version comes from vx.y.z-rcN tags, and the final version should come from vx.y.z tags.
+
+## Vote on the release candidate
+
+The release voting takes place on the Apache celeborn developers list.
+
+- If possible, attach a draft of the release notes with the email.
+- Recommend represent voting closing time in UTC format.
+- Make sure the email is in text format and the links are correct.
+
+Once the vote is done, you should also send out a summary email with the totals, with a subject that looks
+something like __[VOTE][RESULT] Release Apache celeborn ...__
+
+## Finalize the Release
+
+__Be Careful!__
+
+__THIS STEP IS IRREVERSIBLE so make sure you selected the correct staging repository.__
+__Once you move the artifacts into the release folder, they cannot be removed.__
+
+After the vote passes, to upload the binaries to Apache mirrors, you move the binaries from dev directory (this should
+be where they are voted) to release directory. This "moving" is the only way you can add stuff to the actual release
+directory. (Note: only PMC members can move to release directory)
+
+Move the subdirectory in "dev" to the corresponding directory in "release". If you've added your signing key to the
+KEYS file, also update the release copy.
+
+```shell
+build/release/release.sh finalize
+```
+
+Verify that the resources are present in https://downloads.apache.org/incubator/. It may take a while for them to be visible.
+This will be mirrored throughout the Apache network.
+
+For Maven Central Repository, you can Release from the [Apache Nexus Repository Manager](https://repository.apache.org/).
+Log in, open "Staging Repositories", find the one voted on, select and click "Release" and confirm. If successful, it
+should show up under https://repository.apache.org/content/repositories/releases/org/apache/celeborn/ and the same under
+https://repository.apache.org/content/groups/maven-staging-group/org/apache/celeborn/ (look for the correct release version).
+After some time this will be synced to [Maven Central](https://search.maven.org/) automatically.
+
+## Promote the release
+
+### Update Website
+
+Fork and clone [Apache Celeborn website](https://github.com/apache/celeborn-website)
```

Comment:
```suggestion
Fork and clone [Apache Celeborn website](https://github.com/apache/incubator-celeborn-website)
```
 [Apache Celeborn website](https://github.com/apache/celeborn-website) could not visit. PTAL.